octobot_trading/exchanges/connectors/ccxt/ccxt_connector.py

```diff
@@ -245,6 +245,12 @@ def add_options(self, options_dict):
     @ccxt_client_util.converted_ccxt_common_errors
     async def _ensure_auth(self):
         try:
+            # load markets before calling _ensure_auth() to avoid fetching markets status while they are cached
+            with self.error_describer():
+                await self.load_symbol_markets(
```

Comment:
I moved this code here to profit from the `except (octobot_trading.errors.AuthenticationError, ccxt.AuthenticationError)` catch